src/theme/button-link.css.ts

```diff
@@ -23,6 +23,42 @@ export const buttonLinkStyle = recipe({
   },
 })
 
+export const buttonLinkContainerStyle = recipe({
+  base: {
+    display: "flex",
+    alignItems: "center",
+  },
+  defaultVariants: {
+    position: "inline",
+  },
+  variants: {
+    position: {
+      inline: {
+        display: "inline-flex",
+      },
+      left: {
+        justifyContent: "left",
+        // TODO: This might not be suitable for all future variants, adjust at your own disgression
+        width: "100%",
+      },
+      center: {
+        justifyContent: "center",
+        // TODO: This might not be suitable for all future variants, adjust at your own disgression
+        width: "100%",
+      },
+      right: {
+        justifyContent: "right",
+        // TODO: This might not be suitable for all future variants, adjust at your own disgression
+        width: "100%",
+      },
```

Comment:
```suggestion
        justifyContent: "left",
        // TODO: This might not be suitable for all future variants, adjust at your own discretion
        width: "100%",
      },
      center: {
        justifyContent: "center",
        // TODO: This might not be suitable for all future variants, adjust at your own discretion
        width: "100%",
      },
      right: {
        justifyContent: "right",
        // TODO: This might not be suitable for all future variants, adjust at your own discretion
        width: "100%",
      },
```

Reply:
Lol, thanks :D I was really tired yesterday :D 

Weekend can't come soon enough :D